src/global_styling/mixins/_helpers.ts

```diff
@@ -147,10 +147,11 @@ export const useEuiYScroll = ({ height }: _EuiYScroll = {}) => {
 
 export const euiYScrollWithShadows = (
   euiTheme: UseEuiTheme,
-  { height }: _EuiYScroll = {}
+  { height }: _EuiYScroll = {},
+  side?: 'both' | 'start' | 'end'
 ) => `
   ${euiYScroll(euiTheme, { height })}
-  ${euiOverflowShadowStyles(euiTheme, { direction: 'y' })}
+  ${euiOverflowShadowStyles(euiTheme, { direction: 'y', side: side || 'both' })}
```

Comment:
minor code nit/suggestion - use `=` fallback vs `||`
```suggestion
  side: 'both' | 'start' | 'end' = 'both'
) => `
  ${euiYScroll(euiTheme, { height })}
  ${euiOverflowShadowStyles(euiTheme, { direction: 'y', side })}
```

Reply:
Actually sorry on closer look at the API we should likely just extend the `_EuiYScroll` obj. e.g.:

```ts
interface _EuiYScrollWithShadows extends _EuiYScroll {
  side?: 'both' | 'start' | 'end';
}
export const euiYScrollWithShadows = (
  euiTheme: UseEuiTheme,
  { height, side = 'both' }: _EuiYScrollWithShadows = {},
) => `
  ${euiYScroll(euiTheme, { height })}
  ${euiOverflowShadowStyles(euiTheme, { direction: 'y', side })}
```